decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -37,6 +37,9 @@
   # or set it up manually and prevent any ENV manipulation:
   # config.force_ssl = true
 
+  # Enable the service worker. By default is disabled in development and enabled in the rest of environments.
+  config.service_worker_enabled = Rails.env.exclude?("development")
```

Comment:
I was thinking should we allow configuring this through the ENV variables as we can today configure other options in Decidim?

I like the idea that it is always disabled in `development` but I was just thinking if someone wants to disable it in other environments too (for one reason or another).

Reply:
FYI, I started this PR configuring this flag with an ENV variable, but I rejected it because in the Decidim configuration no value is read via ENV vars (although Rails.env is a environment variable). 

If you agree, I can change this to something like: `config.service_worker_enabled = ENV.fetch("SERVICE_WORKER_ENABLED", "true") == "true"`

Reply:
I think it would be best to add to the `secrets.yml`. I know originally it's meant to store only the secrets but today we are already storing much more in that file in Decidim, I guess for convenience reasons. Such as API depth configurations, CSV column separator, throttling period, etc. which are not really secrets but they are defined in `secrets.yml` for convenience.